packages/google_maps_flutter/google_maps_flutter/ios/Classes/JsonConversions.m

```diff
@@ -6,66 +6,163 @@
-  return CLLocationCoordinate2DMake([FLTGoogleMapJsonConversions toDouble:data[0]],
-                                    [FLTGoogleMapJsonConversions toDouble:data[1]]);
++ (CLLocationCoordinate2D)getLocationFromData:(NSArray *)data {
+  return CLLocationCoordinate2DMake([FLTGoogleMapJsonConversions getDoubleFromData:data[0]],
+                                    [FLTGoogleMapJsonConversions getDoubleFromData:data[1]]);
 }
 
-+ (CGPoint)toPoint:(NSArray *)data {
-  return CGPointMake([FLTGoogleMapJsonConversions toDouble:data[0]],
-                     [FLTGoogleMapJsonConversions toDouble:data[1]]);
++ (CGPoint)getPointFromArray:(NSArray *)data {
+  return CGPointMake([FLTGoogleMapJsonConversions getDoubleFromData:data[0]],
+                     [FLTGoogleMapJsonConversions getDoubleFromData:data[1]]);
 }
 
-+ (NSArray *)positionToJson:(CLLocationCoordinate2D)position {
-  return @[ @(position.latitude), @(position.longitude) ];
++ (NSArray *)getJsonFromLocation:(CLLocationCoordinate2D)location {
+  return @[ @(location.latitude), @(location.longitude) ];
 }
 
-+ (UIColor *)toColor:(NSNumber *)numberColor {
++ (UIColor *)getColorFromData:(NSNumber *)numberColor {
   unsigned long value = [numberColor unsignedLongValue];
   return [UIColor colorWithRed:((float)((value & 0xFF0000) >> 16)) / 255.0
                          green:((float)((value & 0xFF00) >> 8)) / 255.0
                           blue:((float)(value & 0xFF)) / 255.0
                          alpha:((float)((value & 0xFF000000) >> 24)) / 255.0];
 }
 
-+ (NSArray<CLLocation *> *)toPoints:(NSArray *)data {
++ (NSArray<CLLocation *> *)getPointsFromData:(NSArray *)data {
   NSMutableArray *points = [[NSMutableArray alloc] init];
   for (unsigned i = 0; i < [data count]; i++) {
     NSNumber *latitude = data[i][0];
     NSNumber *longitude = data[i][1];
-    CLLocation *point =
-        [[CLLocation alloc] initWithLatitude:[FLTGoogleMapJsonConversions toDouble:latitude]
-                                   longitude:[FLTGoogleMapJsonConversions toDouble:longitude]];
+    CLLocation *point = [[CLLocation alloc]
+        initWithLatitude:[FLTGoogleMapJsonConversions getDoubleFromData:latitude]
+               longitude:[FLTGoogleMapJsonConversions getDoubleFromData:longitude]];
     [points addObject:point];
   }
 
   return points;
 }
 
-+ (NSArray<NSArray<CLLocation *> *> *)toHoles:(NSArray *)data {
++ (NSArray<NSArray<CLLocation *> *> *)getHolesFromData:(NSArray *)data {
   NSMutableArray<NSArray<CLLocation *> *> *holes = [[[NSMutableArray alloc] init] init];
   for (unsigned i = 0; i < [data count]; i++) {
-    NSArray<CLLocation *> *points = [FLTGoogleMapJsonConversions toPoints:data[i]];
+    NSArray<CLLocation *> *points = [FLTGoogleMapJsonConversions getPointsFromData:data[i]];
     [holes addObject:points];
   }
 
   return holes;
 }
 
++ (nullable NSDictionary<NSString *, id> *)getJsonFromPosition:(GMSCameraPosition *)position {
```

Comment:
Oops, one thing I missed: these newly-added-to-this-class methods should get new unit tests since they should now be easily unit-testable.